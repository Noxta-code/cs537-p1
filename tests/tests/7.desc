Tie breaking when all k-th words are the same