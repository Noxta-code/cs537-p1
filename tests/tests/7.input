king alpha beta
knight alpha
kind alpha gamma
kangaroo alpha alpha alpha
karate alpha
kind alpha