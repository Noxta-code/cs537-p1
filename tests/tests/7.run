../solution/sortk tests/7.input 2
