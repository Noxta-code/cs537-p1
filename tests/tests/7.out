kangaroo alpha alpha alpha
karate alpha
kind alpha
kind alpha gamma
king alpha beta
knight alpha
